staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -864,6 +864,10 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	EnableDebugFlagsHandler *bool `json:"enableDebugFlagsHandler,omitempty"`
+	// SeccompDefault enables the use of `RuntimeDefault` as the default seccomp profile for all workloads.
```

Comment:
note this required the SeccompDefault feature gate?

Reply:
Yep, added a comment as suggested :+1: